src/sentry/api/serializers/models/alert_rule.py

```diff
@@ -46,7 +46,7 @@ def serialize(self, obj, attrs, user):
             "triggers": attrs.get("triggers", []),
             "includeAllProjects": obj.include_all_projects,
             "dateModified": obj.date_modified,
-            "dateAdded": obj.date_added,
+            "dateCreated": obj.date_added,
```

Comment:
Would we want to update the db model as well to match?

Reply:
This isn't a change in the database - only in how we name the field when serializing it.

The database remains as date_added.

Reply:
For whatever reason, we almost always call this field `date_added` internally, and `dateCreated` externally

Reply:
Ah ok 👍